content/en/cloudcraft/getting-started/datadog-integration.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Datadog Integration
+---
+
+## Overview
+
+The integration between Datadog and Cloudcraft provides users with a streamlined workflow for monitoring and visualizing their cloud infrastructure.
+
+By leveraging Datadog’s powerful monitoring platform, users can log in to Cloudcraft with their Datadog account, move seamlessly from any resource in Cloudcraft to the most relevant views in Datadog, and automatically synchronize cloud accounts that have already been configured in Datadog for use in Cloudcraft.
+
+## Datadog Single Sign-On (SSO)
+
+Cloudcraft allows users to sign up and log in using their Datadog account. This integration offers a unified experience, linking your Datadog monitoring data seamlessly with your Cloudcraft architecture diagrams.
+
+### Signing up with Datadog SSO
```

Comment:
```suggestion
### Sign up with Datadog SSO
```
Just making headers consistent, you can also use "ing" in all headers if you prefer, but the simple present tense can be slightly better for SEO.